homeassistant/components/homekit_controller/sensor.py

```diff
@@ -6,9 +6,19 @@
 HUMIDITY_ICON = 'mdi:water-percent'
 TEMP_C_ICON = "mdi:thermometer"
 BRIGHTNESS_ICON = "mdi:brightness-6"
+CO2_ICON = "mdi:periodic-table-co2"
 
 UNIT_PERCENT = "%"
 UNIT_LUX = "lux"
+UNIT_CO2 = "ppm"
+
+
+ENTITY_TYPES = {
```

Comment:
This will need to go at the bottom of the file like in binary_sensor.py as thse classes won't exist until they are parsed after this. Probably best to move `async_setup_platform` and `async_setup_entry` like binary_sensor too.